README.md

```diff
@@ -21,9 +26,31 @@
 * Sustained disruptions.
 * Violence, threats of violence or violent language.
 
-# Enforcement
+## Enforcement
 * Understand that speech and actions have consequences, and unacceptable behavior will not be tolerated.
 * If you are the subject of, or witness to any violations of this Code of Conduct, please **contact any of the slack administrators**
 * If violations occur, organizers will take any action they deem appropriate for the infraction, up to and including expulsion.
 
 Portions derived from the [Web Community Leads UK and IE](https://webcommunityukie.github.io/), [Samsung Internet Slack Group](https://samsunginter.net/coc), [Slack Community Code of Conduct](https://api.slack.com/docs/community-code-of-conduct), [Django Code of Conduct](https://www.djangoproject.com/conduct/), [The Citizen Code of Conduct](http://citizencodeofconduct.org/), [The Rust Code of Conduct](https://www.rust-lang.org/en-US/conduct.html) and [The Ada Initiative](https://adainitiative.org/2014/02/18/howto-design-a-code-of-conduct-for-your-community/) under a Creative Commons [Attribution-ShareAlike](https://creativecommons.org/licenses/by-sa/3.0/) license.
+
+---
+
+# Guidelines
+These are some of the ways we do our best to implement our code of conduct.
+
+## Slack Etiquette
+
+### Misc
+
+* Tangents or off-topic conversations should be moved to threads or a relevant channel. Someone wise once told me "channels are free" - feel free to start a channel to discuss a topic* if several people are interested in it.
```

Comment:
I think this should say "Someone wise once *said*" rather than "once *told me*" which makes it more inclusive to the community rather than specific to a single person